test/forms/labels/error-label.test.js

```diff
@@ -1,15 +1,22 @@
 import React from 'react'
-import { shallow } from 'enzyme'
 import { ErrorLabel } from '../../../src'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
 
 test('passes class to span element correctly', () => {
-  const wrapper = shallow(<ErrorLabel className="small">Error!</ErrorLabel>)
-  expect(wrapper.hasClass('error-message')).toBe(true)
-  expect(wrapper.hasClass('small')).toBe(true)
+  render(<ErrorLabel className="small">Error!</ErrorLabel>)
+  const label = screen.getByText('Error!')
+  expect(label).toHaveClass('error-message', 'small')
 })
 
-test('passes extra props to span element', () => {
-  const onClick = () => 'More info'
-  const wrapper = shallow(<ErrorLabel onClick={onClick}>Error!</ErrorLabel>)
-  expect(wrapper.props().onClick).toBe(onClick)
+test('passes extra props to span element', async () => {
+  let count = 0
+  const onClick = () => count++
+  const user = userEvent.setup()
+
+  render(<ErrorLabel onClick={onClick}>Error!</ErrorLabel>)
+
+  await user.click(screen.getByText('Error!'))
+
+  expect(count).toBe(1)
```

Comment:
This is interesting. I haven't seen that before, but it seems to cover the test case.
Wouldn't passing a data attribute be simpler though?

Reply:
Yeah. This was written prior to me taking over the PR. I can modify it